package.json

```diff
@@ -34,7 +34,8 @@
     "test-browser": "node tests/browser",
     "test-cli": "nodeunit tests/cli.js",
     "test-node": "npm run test-unit && npm run test-cli && npm run test-regression",
-    "test-regression": "nodeunit tests/regression",
+    "test-regression": "nodeunit tests/regression && node bin/jshint src",
+    "test-rhino": "java -jar tests/regression/libs/rhino-1.7.7.jar -opt -1 dist/jshint-rhino.js src",
```

Comment:
Does this command only run the packaged JSHint file? Is it possible to run the tests from the `tests/` dir?